website/sidebars.json

```diff
@@ -108,6 +108,7 @@
         "ids": [
           "developers/developer-reference/elrond-serialization-format",
           "developers/developer-reference/elrond-wasm-annotations",
+          "developers/developer-reference/elrond-wasm-modules",
```

Comment:
in the elrond-wasm-module, there is a paragraph
```
Conclusion
We hope this module system will make it a lot easier to write maintainable smart contract code, and even reusable modules.

If you want to take a look at some more examples, or even just modules you can use, take a look here: https://github.com/ElrondNetwork/elrond-wasm-rs/tree/master/contracts/modules
```
which points to a dead link. Can you please fix that?